com.jianguoyun.Nutstore.yaml

```diff
@@ -0,0 +1,81 @@
+app-id: com.jianguoyun.Nutstore
```

Comment:
I'm having bit of trouble loading this `jianguoyun.com` domain/website on a browser. 

It seems to be blocking connections from outside China?

Reply:
I can access https://www.jianguoyun.com/ from Los Angeles.

Reply:
Yea seems blocked in my country, I could access it via VPN to US.